ansible_base/authentication/authenticator_plugins/ldap.py

```diff
@@ -123,7 +123,7 @@ def validate_ldap_filter(value: Any, with_user: bool = False) -> bool:
             # We only need to check with_user at the top of the recursion stack
             validate_ldap_filter(f'({sub_filter})', with_user=False)
         return
-    raise ValidationError(_('Invalid filter: %s') % value)
+    raise ValidationError(_('Invalid filter: %s' % value))
```

Comment:
This doesn't feel correct to me. If I understand right, this will do the string interpolation _before_ `_()` meaning that the fully interpolated string is what will land in the translation files.